microlith/scripts/collect-information.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Don't set euo pipefail - we want this script to be as resilient as possible
+set -x
+
+echo "Starting collect-information.sh..."
+
+tmpdir=${TEMPORARY_DIRECTORY:-$(mktemp -d)}
+exec &> >(tee -a "$tmpdir/collect-information.sh.log")
+
+cp /etc/couchbase-cluster-monitor-release.txt "$tmpdir"
+cp /etc/cmos-release.txt "$tmpdir"
+
+# Environment
+env > "$tmpdir/env.txt"
+
+# Copy over all logs
+mkdir -p "$tmpdir/logs"
+cp /logs/* "$tmpdir/logs/"
+
+# Do not copy /var/log/nginx/*, because they get mapped to stdout/stderr and cp will hang forever
```

Comment:
Do we want to break these links?

Reply:
Possibly? Depends on if it's the nginx entrypoint doing it, in which case we'd have to replace it